### PR TITLE
fix: return ID on create

### PR DESCRIPTION
**Pull request by zepatrik**
Currently the postgres and cockroach dialects have the `id` field hard coded instead of respecting the `db:"whatever"` tag of the `ID` field. This PR fixes this problem.

### CODE REVIEW

File: dialect_cockroach.go [hunk 1]
```diff
@@ -68,30 +68,28 @@ func (p *cockroach) Create(s store, model *Model, cols columns.Columns) error {
...
+		id := map[string]interface{}{}
+		err = stmt.QueryRow(model.Value).MapScan(id)
 		if err != nil {
 			if err := stmt.Close(); err != nil {
 				return errors.WithMessage(err, "failed to close statement")
 			}
 			return err
 		}
-		model.setID(id.ID)
+		model.setID(id[model.IDField()])
```
Comment (aeneasr) [inline dialect_cockroach.go:92]: Is it safe to assume that this entry always exists?

Comment (zepatrik) [reply dialect_cockroach.go:92]: This is basically the dynamic version of https://github.com/gobuffalo/pop/pull/590/files#diff-bbab701ece62d45ddcad2772dbb0d7feL71-R71

Comment (zepatrik) [reply dialect_cockroach.go:92]: I mean we are always `... returning {{ model.IDField() }}` in the sql query so I guess it is safe to assume it exists.

**Review by aeneasr — APPROVED** [review]

**Comment by stanislas-m** [comment]
Thanks!

**Pull request merged**